abstract/abstract.md

```diff
@@ -0,0 +1,12 @@
+---
+title: 'Geophysics in a changing climate'
+abstract: |
+ It is clear that the geosciences have an important role to play in working to understand and mitigate the societal impacts of the climate crisis. Locating critical minerals, monitoring geologic storage of CO2, managing groundwater, and characterizing changes to permafrost are all applications where geophysical data can provide insights. These applications raise interesting scientific questions about how to combine petrophysical, geologic, geochemical, and additional geophysical data sets to improve our ability to produce useful models of the subsurface. The next significant advancements will undoubtedly involve methodological improvements in inversions and machine learning, but importantly will require a more interdisciplinary approach, where the methods we design can be used to test and revise hypotheses specific to a given geologic context. The growth and adoption of open-source software and data standards is a positive step. Within the SimPEG community, we have seen many examples of how working in a modular, interoperable framework enables ideas to rapidly be prototyped and extended. Working in an open ecosystem allows us to more readily connect with methods in other fields, including geologic modelling and machine learning. 
+
+ Although geophysics has an important role to play in mitigating climate change and its impacts, we are at risk of continuing to be a “field in decline.” Geosciences as a whole have a severe diversity problem, and with the strong association of geophysics with hydrocarbons deterring many students, we are missing out on the talents and ideas of many. In this talk, I look forward to opening up what I hope will be continuing discussions about growing our community and broadening the scientific and societal contributions we can make. 
+---
+
+![thumbnail](thumbnail.png)
+
+[YouTube Video](https://www.youtube.com/watch?v=B6ZaiwcUaFA)
```

Comment:
Here, we can embed videos using the following, instead of just linking it (e.g. this shows up in the myst docs here: https://mystmd.org/guide/figures#youtube-videos). 

In the cases where we have an embedded video, lets remove the thumbnail image here and just have the video too.  

```suggestion
:::{iframe} https://www.youtube.com/watch?v=B6ZaiwcUaFA
:width: 100%
:::
```

Reply:
I tried with 'iframe' command but it shows up like this, maybe i am missing something here? 
<img width="811" alt="Screenshot 2024-07-21 at 1 06 07 PM" src="https://github.com/user-attachments/assets/56ed8158-a2a0-45d5-b534-e1a1d9cb8d0f">

Reply:
I see the same thing. Feel free to make the change and include the iframe and lets see if it shows up in the curvenote preview. If it does, then we are good to go. Otherwise, we can get some help from Rowan and Steve on this.